### PR TITLE
Use named volume for /master on Node

### LINKED ISSUE #438: Do not overwrite Tendermint configs during initialization
Now `tendermint init` is running every time the Node started. It could lead to some information or configuration being lost.

The same considerations may be applied to Worker initialization: Node always overwrites Worker's config, even if it's already present.

### LINKED ISSUE #452: import-export configs from nodes
This will give functionality to transfer or store configs, keys, and settings of nodes.

### PR DESCRIPTION
**Pull request by folex**
fixes #438
fixes #452 , since all settings are in `~/.fluence` directory on host. So you should be able to import/export settings, data and keys.

In effect, everything saved by workers and tendermint will be also persisted. That's because they use `--volumes-from`, and then basically write all data to `/master/*/*`

### CODE REVIEW

File: node/src/it/scala/fluence.node/DockerSetup.scala [hunk 1]
```diff
@@ -34,32 +36,55 @@ trait DockerSetup extends OsSetup {
...
+      case _ => path
+    }
+
+    Resource.make(
+      Sync[F]
+        .delay(macOsHack(Files.createTempDirectory("testvolume").toString))
+    )(
+      f =>
+        Sync[F].delay {
+          println(s"removing temp file $f")
```
Comment (DieMyst) [inline node/src/it/scala/fluence.node/DockerSetup.scala]: println

File: node/src/it/scala/fluence.node/DockerSetup.scala [hunk 2]
```diff
@@ -34,32 +36,55 @@ trait DockerSetup extends OsSetup {
...
+    def macOsHack(path: String): String = getOS match {
+      case "mac" => path.replaceFirst("^/var/folder", "/private/var/folder")
+      case _ => path
+    }
+
+    Resource.make(
+      Sync[F]
+        .delay(macOsHack(Files.createTempDirectory("testvolume").toString))
+    )(
+      f =>
```
Comment (DieMyst) [inline node/src/it/scala/fluence.node/DockerSetup.scala]: what is `f`?

File: tools/deploy/scripts/compose.sh [hunk 3]
```diff
@@ -159,14 +174,17 @@ function export_arguments()
...
         export PARITY_RESERVED_PEERS='../config/reserved_peers.txt'
-        export PARITY_STORAGE='~/.parity/'
+        export PARITY_STORAGE="$HOME/.parity/"
         export PARITY_ARGS='--config dev-insecure --jsonrpc-apis=all --jsonrpc-hosts=all --jsonrpc-cors="*" --unsafe-expose'
     else
         echo "Deploying for $CHAIN chain."
         export PARITY_ARGS='--light --chain '$CHAIN' --jsonrpc-apis=all --jsonrpc-hosts=all --jsonrpc-cors="*" --unsafe-expose --reserved-peers=/reserved_peers.txt'
     fi
+
+    export FLUENCE_STORAGE="$HOME/.fluence/"
```
Comment (DieMyst) [inline tools/deploy/scripts/compose.sh]: this argument is duplicated

**Comment by folex** [comment]
I'll address comments in #460

**Pull request merged**